templates/cisco_ios_show_interface_switchport.template

```diff
@@ -0,0 +1,20 @@
+Value Required INTERFACE (\S+)
+Value SWITCHPORT (\S+)
+Value ADMIN_MODE (.+)
+Value OPER_MODE (.+)
+Value DTP (\S+)
+Value ACCESS_VLAN (\S+)
+Value NATIVE_VLAN (\S+)
+Value VOICE_VLAN (\S+)
+Value TRUNKING_VLANS (\S+)
+
+Start
+  ^Name: ${INTERFACE}
+  ^Switchport: ${SWITCHPORT}
+  ^Administrative Mode: ${ADMIN_MODE}
+  ^Operational Mode: ${OPER_MODE}
+  ^Negotiation of Trunking: ${DTP}
+  ^Access Mode VLAN: ${ACCESS_VLAN}
+  ^Trunking Native Mode VLAN: ${NATIVE_VLAN}
+  ^Voice VLAN: ${VOICE_VLAN}
+  ^Trunking VLANs Enabled: ${TRUNKING_VLANS} -> Record
```

Comment:
Do you mind changing from static spaces to using `\s*` or `\s+` within the state?

Do you mind adding the following to the end of `Start` state:
```
  ^\s*$$
  ^. -> Error
```

Note: This may cause failures if not all lines of output are accounted for, which appears will be the case.